client/src/components/Column/Column.js

```diff
@@ -24,13 +22,10 @@ function Column({
 
       {questions
         .filter((question) => question.category === category)
-        .map((question, index) => (
+        .map((question) => (
           <QuestionButton
-            disabled={isDisabled}
-            key={currQuestionID}
-            onDisabled={() => {
-              setIsDisabled(!isDisabled);
-            }}
+            disabled={answeredQuestions[question.id]}
+            key={question.id}
```

Comment:
Remember that `currQuestionId` is going to be the same for all columns, because only 1 question is selectable at once. Instead, you want your key to be some unique property of this specific question button. The most sensible choice here is just the ID of the question this button represents.